src/config.js

```diff
@@ -51,6 +53,7 @@ class Config {
     this.runtimeMetrics = String(runtimeMetrics) === 'true'
     this.experimental = {}
     this.reportHostname = String(reportHostname) === 'true'
+    this.scope = process.env.DD_CONTEXT_PROPAGATION === 'false' ? scopes.NOOP : scope
```

Comment:
Can we invert the condition so it's:
`this.scope = process.env.DD_CONTEXT_PROPAGATION === 'true' ? scope : scopes.NOOP`

Reply:
`DD_CONTEXT_PROPAGATION` is not set by default, and is only used to disable it so its only possible value is always `false`.